WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -1176,6 +1176,11 @@ - (ReaderSiteTopic *)findSiteTopicWithFeedID:(NSNumber *)feedID
     return (ReaderSiteTopic *)[results firstObject];
 }
 
+- (ReaderSiteTopic *)siteTopicForSiteID:(NSNumber *)siteID
+{
+    return [self findSiteTopicWithSiteID:siteID] ?: [self findSiteTopicWithFeedID:siteID];
```

Comment:
These methods will return a topic that is already synced to the device, but it seems likely that the user won't already be following many likers and we'd need to fetch site or feed info in those cases. 
`siteTopicForSiteWithID:isFeed:success:failure:` is set up to do this. Maybe the existing method would be a better use case?  
If it helps, the siteIDs we get from the likes endpoints should never be feedIDs, just siteIDs. (I think. Pretty sure Jetpack sites are siteIDs.  FeedIDs would only be external sites crawled and stored in the feedbag).